README.md

```diff
@@ -1,6 +1,7 @@
 # authn_simple
 
-[![Build Status](https://travis-ci.org/icatproject/authn.simple.svg?branch=master)](https://travis-ci.org/icatproject/authn.simple)
+[![Build Status](https://github.com/icatproject/authn.simple/workflows/CI%20Build/badge.svg?branch=master)](https://github.com/icatproject/authn.simple/actions?query=workflow%3A%22CI+Build%22
+)
```

Comment:
Closing bracket is on a new line (not that this matters functionally, I'm just nitpicking).